nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/_status-history.component-theme.scss

```diff
@@ -0,0 +1,63 @@
+/*!
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+@mixin theme($material-theme, $canvas-theme) {
+    // Get the color config from the theme.
+    $color-config: mat.get-color-config($material-theme);
+    $canvas-color-config: mat.get-color-config($canvas-theme);
+
+    // Get the color palette from the color-config.
+    $primary-palette: map.get($color-config, 'primary');
+    $canvas-primary-palette: map.get($canvas-color-config, 'primary');
+
+    // Get hues from palette
+    $primary-palette-700: mat.get-color-from-palette($primary-palette, 700);
+    $canvas-primary-palette-900: mat.get-color-from-palette($canvas-primary-palette, 900);
+    $canvas-primary-palette-A100: mat.get-color-from-palette($canvas-primary-palette, 'A100');
+    $canvas-primary-palette-A700: mat.get-color-from-palette($canvas-primary-palette, 'A700');
+
+    #status-history-chart-container,
+    #status-history-chart-control-container {
+        background-color: $canvas-primary-palette-900;
+
+        .axis path,
+        .axis line {
+            fill: none;
+            stroke: $canvas-primary-palette-A100;
```

Comment:
This translates into a transparent stroke for the chart axis lines (invisible).

New UI, no x or y axis:
<img width="833" alt="Screenshot 2024-01-30 at 8 34 06 AM" src="https://github.com/apache/nifi/assets/713866/05fd832e-5051-458c-b0b5-0149dd203944">


Old UI, both x and y axis
<img width="765" alt="Screenshot 2024-01-30 at 8 34 33 AM" src="https://github.com/apache/nifi/assets/713866/c3dd91cd-d1e3-4435-a061-c942d908db97">

Reply:
Let's just come back to this in the future. This PR is too big to keep rebasing over and over.